contracts/RoyaltyFeeManager.sol

```diff
@@ -17,19 +19,27 @@ contract RoyaltyFeeManager is
     Initializable,
     OwnableUpgradeable
 {
+    using RoyaltyFeeTypes for RoyaltyFeeTypes.FeeInfoPart;
+
     // https://eips.ethereum.org/EIPS/eip-2981
     bytes4 public constant INTERFACE_ID_ERC2981 = 0x2a55205a;
 
     IRoyaltyFeeRegistry public royaltyFeeRegistry;
+    IRoyaltyFeeRegistryV2 public royaltyFeeRegistryV2;
 
     /**
      * @notice Initializer
      * @param _royaltyFeeRegistry address of the RoyaltyFeeRegistry
+     * @param _royaltyFeeRegistryV2 address of the RoyaltyFeeRegistryV2
      */
-    function initialize(address _royaltyFeeRegistry) public initializer {
+    function initialize(
+        address _royaltyFeeRegistry,
+        address _royaltyFeeRegistryV2
+    ) public initializer {
```

Comment:
i'm not sure if its safe to change the `initialize` method for an already deployed contract or if I should just create a setter method instead

Reply:
A proxy only uses its storage, all the logic is made by the implementation, so all functions, parameters, events etc are handled by the implementation itself.
So it's safe :)